### DISCUSSION TITLE
incorrect slop of runway and airport

### DISCUSSION THREAD

**Discussion by Pilotmk3** [kubilus1/autoortho#302] [General]
I have downloaded Asia region and am getting incorrect slop of runway and airport with lots of ups and down, is that due to autoortho ? is there any way to solve it ?

**Comment by d41k4n** [comment]
It is due to the way how AO scenery is prepared and distributed through the installer.

To fix your issue you would have to provide more info (e.g. screenshots, airport ICAO) as it could have multiple root causes. E.g. it could be due to faulty/missing OSM data and/or anomalies in the raster elevation data used to prepare the scenery with Ortho4XP.

Alternatively, you could try to patch the mesh locally yourself with [MUXP](https://forums.x-plane.org/index.php?/files/file/67230-mesh-updater-x-plane-muxp/).

**Reply by Pilotmk3** [reply]
sure, here is screen shot of airports 
VOHS (hydrabad india)
![20230930224459_1](https://github.com/kubilus1/autoortho/assets/141163254/6eb87ac2-229a-4f1d-8c2e-8e6f7f717736)
![20230930224101_1](https://github.com/kubilus1/autoortho/assets/141163254/6af4180d-d745-4278-a7fc-ce22793c8ea5)
![20230930224121_1](https://github.com/kubilus1/autoortho/assets/141163254/26134eef-c25b-424d-a61b-b0037a379e57)
![20230930224355_1](https://github.com/kubilus1/autoortho/assets/141163254/ee4844cd-2448-4c5f-b34c-8368dac01969)

VARK (rajkot india)
![20230930225142_1](https://github.com/kubilus1/autoortho/assets/141163254/80f926b2-3494-4b4d-9a56-e3e8447cc28f)


and here is log file 
[Log.txt](https://github.com/kubilus1/autoortho/files/12775240/Log.txt)

and scenery.ini here 
[scenery.txt](https://github.com/kubilus1/autoortho/files/12775241/scenery.txt)

**Comment by d41k4n** [comment]
IMO this is something that would have to be fixed using a different elevation data source and/or different airport smoothing pix parameter. This will require building and testing the tile in Ortho4XP and once the correct build data source and/or parameters are confirmed this will have to be updated in the AO scenery authoring step so the updated DSF can be distributed with the installer.

**Reply by Pilotmk3** [reply]
okay i will try with Ortho4XP, thankyou for help. 
and also can you help me regarding no buildings in scenery.
![20230930230800_1](https://github.com/kubilus1/autoortho/assets/141163254/b8f34982-4f96-4b7c-9b65-a7917de0256e)

**Reply by d41k4n** [reply]
> and also can you help me regarding no buildings in scenery.

This is most likely due to a wrong installation (wrong ordering of entries in scenery_packs.ini) and unrelated to this issue and you've already asked that same question [here](https://github.com/kubilus1/autoortho/discussions/232#discussioncomment-7152744).

I suggest to search the discussions [here](https://github.com/kubilus1/autoortho/discussions) and the [dedicated forum on x-plane.org](https://forums.x-plane.org/index.php?/forums/forum/802-autoortho-streaming-ortho-imagery-for-x-plane/) for possible support with the installation and post any open questions there.
